src/lockfile/lockfile.go

```diff
@@ -0,0 +1,139 @@
+package lockfile
```

Comment:
Move this package to a new vendor directory and then you don't need to change the $GOPATH.

```
src/
├── vendor
│   └── lockfile
│       └── lockfile.go

```

Reply:
That doesnt work so easy. It seems like the vendor folder needs the additional program govendor to be installed. When i understand it correctly, the vendor folder is an alternative to the normal $GOPATH. Using it with local packages is possible, but i dont think it is so good to do.
I also can upload it to a new github repo and import it from there, if you want.

I would say, keeping it, as it is currently is, will be the easiest way, but i can set up, other things, if you want :)

Reply:
Vendor directory is standard as of Go 1.6, so dependencies should be placed in there and they will work with the standard go commands like `go build`.

When using the go tools such as go build or go run, they first check to see if the dependencies are located in ./vendor/. If so, use it. If not, revert to the $GOPATH/src/ directory.

Reply:
On my environment with Go 1.9.1 on ubuntu 16.04 it cant compile anymore, cause the dependensies cant be found:
```
manager/src$ go build
mod_Mods.go:10:2: cannot find package "lockfile" in any of:
        /usr/local/go/src/lockfile (from $GOROOT)
        /home/knox/Desktop/Programme/GO/factorio-server-manager/manager/src/lockfile (from $GOPATH)
        /home/knox/go/src/lockfile
```

Do i need to add a vendor,json to make it work, or does it work without?

Reply:
No it should just work with the file structure I posted.  I tested on my side and I can build it fine with the vendor directory and I don't have any extra dependency management tools installed.

My src/ directory looks like this:

```
.
├── auth.go
├── factorio_config.go
├── factorio_server.go
├── factorio_server_linux.go
├── factorio_server_windows.go
├── gamelog.go
├── handlers.go
├── main.go
├── mod_modInfo.go
├── mod_modpacks.go
├── mod_Mods.go
├── mod_modSimple.go
├── mods.go
├── mods_handler.go
├── mods_test.go
├── rcon.go
├── routes.go
├── saves.go
├── vendor
│   └── lockfile
│       └── lockfile.go
├── wsclient.go
└── wsroutes.go
```

Reply:
I misunderstood the GOPATH and read more things about it. It is working in my environment too and i updated the fle.

Reply:
Nice!